src/plugins/telemetryTable/collections/TableRowCollectionSpec.js

```diff
@@ -0,0 +1,58 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+import TableRowCollection from "./TableRowCollection.js";
+
+describe("Table rows constructor", function () {
```

Comment:
Thank you for contributing some tests. We do tend to test at a higher level of abstraction than this, and we tend not to test classes in isolation with the possible exception of API classes.

Instead we tend to test at a [plugin level](https://github.com/nasa/openmct/blob/6bdb8c9e1c9421c150a3db48eb52ff5d2d130b11/src/plugins/telemetryTable/pluginSpec.js#L296), or at a [UI level](https://github.com/nasa/openmct/blob/master/e2e/tests/functional/plugins/telemetryTable/telemetryTable.e2e.spec.js).

As such, I think it's ok to forgo tests for this PR since we already [have automated tests that cover filtering](https://github.com/nasa/openmct/blob/6bdb8c9e1c9421c150a3db48eb52ff5d2d130b11/src/plugins/telemetryTable/pluginSpec.js#L296). If you wanted to expand those tests, then that's great too.

Reply:
No problem @akhenry .The linting problems have been fixed and the testing for that class just deleted.